joomla/joomla-cms
# Lang string JGLOBAL_PRINT_TITLE outputs invalid HTML

### Testing Instructions
- See title attribute of print button link in frontend article view (page source code)
```
title="Print article < Title of article >"
```
=> invalid HTML

After patch
```
title="Print article &lt; Title of article &gt;"
```
